maintenance/maintenance_test.go

```diff
@@ -0,0 +1,68 @@
+package maintenance
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/bitrise-io/go-utils/sliceutil"
+)
+
+type ResponseBody []DirectoryEntry
+type DirectoryEntry struct {
+	Name string `json:"name"`
+}
+
+var stacks = []string{
```

Comment:
The feature to select stack in bitrise-init is not merged yet, see https://github.com/bitrise-io/bitrise-init/pull/164/files#r341987533. Can merge this if we update the task https://bitrise.atlassian.net/browse/TOOL-134 to update maintenance test.